internal/info/info_bench_test.go

```diff
@@ -0,0 +1,119 @@
+package info
+
+import (
+	"testing"
+)
+
+func BenchmarkString(b *testing.B) {
+	Init("benchmark")
+	b.ReportAllocs()
+	b.ResetTimer()
+	b.RunParallel(func(pb *testing.PB) {
+		for pb.Next() {
+			if s := String(); s == "" {
+				b.Error("String return empty string")
+			}
+		}
+	})
+}
+
+func BenchmarkGet(b *testing.B) {
+	sn := "benchmark"
+	Init(sn)
```

Comment:
same
https://github.com/vdaas/vald/pull/1093/files#r600168339

Reply:
replied on that thread